ci/_init.sh

```diff
@@ -15,15 +15,22 @@ fi
 export DRIVER_NAME=libsnowflakeclient
 
 # Build images
-BUILD_IMAGE_VERSION_X64=1
+BUILD_IMAGE_VERSION_X64=2
```

Comment:
let's change also `Dockerfile.libsnowflakeclient-centos7-default-test` to point to the new image